common/directoryBlock/dbInfo/dbInfo.go

```diff
@@ -10,11 +10,12 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
+// DirBlockInfo holds the information for a factom directory block
 type DirBlockInfo struct {
 	// Serial hash for the directory block
 	DBHash    interfaces.IHash
```

Comment:
missing json tags

Reply:
added, please check to make sure they are ok